Neos.Fusion/Classes/Core/FusionDefaultContextVariables.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Fusion\Core;
+
+use Neos\Utility\Arrays;
+
+/**
+ * Variables from configuration that should be set in the context by default.
+ * For example Eel helper definitions
+ */
+final class FusionDefaultContextVariables
```

Comment:
Im not totally happy with the naming here.

It should ideally reflect that these are the variables / options / metadata / properties ... that are valid for this specific runtime instance. They are not cached and "should ideally not be overridden - at least conceptional". Their purpose is to hold eel helpers or things like `request` and the upcoming editPreviewMode.

They are different to the variables like "node", as they can be set at runtime.

But they are similar to "node" and other variables, as they can be accessed like regulary in eel or via `$runtime->getContextVariable`

Reply:
@grebaldi had the idea to name it `FusionGlobals` and i really like it :D